samza-core/src/main/java/org/apache/samza/coordinator/JobCoordinator.java

```diff
@@ -86,4 +82,29 @@
    * @return instance of JobModel that describes the partition distribution among the processors (and hence, tasks)
    */
   JobModel getJobModel();
+
+  /**
+   * Returns the identifier assigned to the processor that is local to the instance of StreamProcessor.
```

Comment:
Can you document a line here about what is expected of other JobCoordinator implementations?  When it should be overriden?